lib/watch-files.js

```diff
@@ -1,17 +1,42 @@
 const chokidar = require('chokidar');
-const { isEmpty } = require('omnibelt');
+const { isEmpty, forEachSerialP } = require('omnibelt');
 const { logError, logResult, loadConfig, unlockConfig } = require('./utils');
 const paramsByCommand = require('./get-upload-params');
 const getUploader = require('./get-uploader');
 const path = require('path');
 
+const queueInfo = {
+  queue: [],
+  queueData: new Map(),
+  isRunning: false
+};
+
+setInterval(async () => {
+  const { queue, queueData } = queueInfo;
+  let { isRunning } = queueInfo;
+  logResult('upload', 'processing queue', 'magentaBright');
```

Comment:
I would put this in the forEachSerialP function and add the file name you are about to upload.